db/models/invoice.go

```diff
@@ -22,8 +22,8 @@ type Invoice struct {
 	DestinationCustomRecords map[uint64][]byte `json:"custom_records,omitempty"`
 	RHash                    string            `json:"r_hash"`
 	Preimage                 string            `json:"preimage" bun:",nullzero"`
-	Internal                 bool              `json:"-" bun:",nullzero"`
-	Keysend                  bool              `json:"keysend" bun:",nullzero"`
+	Internal                 bool              `json:"-" bun:",notnull,default:false"`
+	Keysend                  bool              `json:"keysend" bun:",notnull,default:false"`
```

Comment:
they still use nullzero:
https://bun.uptrace.dev/guide/models.html#default

`nullzero,notnull,default:false` ?

I think if `""` (blank) is set then bun converts it to `NULL` which then uses the default of `false`.